frontend/src/components/dashboard/minesHomePage/Dashboard.js

```diff
@@ -450,7 +451,7 @@ export class Dashboard extends Component {
               <AuthorizationWrapper permission={Permission.ADMIN}>
                 <AddButton
                   onClick={(event) =>
-                    this.openModal(event, this.handleSubmit, ModalContent.CREATE_MINE_RECORD)
+                    this.openModal(event, this.handleSubmit, ModalContent.CREATE_MINE_RECORD, true)
                   }
```

Comment:
the `true` here isn't needed. 

Instead, you can just initiate the prop inside the openModal function like so: 
```
openModal(event, onSubmit, title) {
    const handleDelete = () => {};
    event.preventDefault();
    this.props.openModal({
      props: {
        handleDelete,
        onSubmit,
        title,
        isCreated: true,
      },
      content: modalConfig.MINE_RECORD,
    });
```